src/clientmetricreport/ClientMetricReport.ts

```diff
@@ -162,7 +162,10 @@ export default class ClientMetricReport {
       transform: this.identityValue,
       type: SdkMetric.Type.VIDEO_AVAILABLE_SEND_BANDWIDTH,
     },
-    currentRoundTripTime: { transform: this.identityValue, type: SdkMetric.Type.STUN_RTT_MS },
+    currentRoundTripTime: {
+      transform: this.secondsToMilliseconds,
```

Comment:
The `type` is named as `STUN_RTT_MS`, which means the metrics should be converted to millisecond based.